### PR TITLE
fixes emagging exofabs

### LINKED ISSUE #11967: Can't emag exofabs (to gain access)
(WEB REPORT BY: sonixapache REMOTE: 172.93.109.202:7777)

> Revision (Should be above if you're viewing this from ingame!)
> 777a7453ab5486dfce43b02b3f9b3dd25b881973
> 
> General description of the issue
> Nothing happens when you emag an exofab.
> 
> What you expected to happen
> Emag exofab, get access
> 
> What actually happened
> slap emag onto exofab, no effect, UNAUTHORISED ACCESS DETECTED

### PR DESCRIPTION
**Pull request by Intigracy**
fixes #11967

### CODE REVIEW

File: code/modules/research/fabricators.dm [hunk 1]
```diff
@@ -81,7 +82,7 @@
 
 /obj/machinery/r_n_d/fabricator/emag()
 	sleep()
-	if(!research_flags &ACCESS_EMAG)
```
Comment (PJB3005) [inline code/modules/research/fabricators.dm:84]: Does this even work? It seems like the check will just do `0 & ACCESS_EMAG` which is always false.

Comment (Intigracy) [reply code/modules/research/fabricators.dm:84]: Yes.
ACCESS_EMAG is a research flag for whatever reason, which the subtype for exofab has.

Comment (Intigracy) [reply code/modules/research/fabricators.dm:84]: Unless you mean the lack of parenthesis, in which case iunno

Comment (PJB3005) [reply code/modules/research/fabricators.dm:84]: Yes, that's the exact problem. The `!` is above `&`.

Comment (Intigracy) [reply code/modules/research/fabricators.dm:84]: fixed btw

**Comment by Manfreedo** [comment]
Please also close https://github.com/d3athrow/vgstation13/issues/9054

**Pull request merged**